nexus/db-queries/src/db/queries/region_allocation.rs

```diff
@@ -117,8 +117,14 @@ pub fn allocation_query(
 
     let seed = seed.to_le_bytes().to_vec();
 
-    let size_delta =
+    // The Crucible Agent's current reservation factor
+    const RESERVATION_FACTOR: f64 = 1.25;
+
+    let requested_size: u64 =
         params.block_size * params.blocks_per_extent * params.extent_count;
+    let size_delta: u64 =
+        (requested_size as f64 * RESERVATION_FACTOR).round() as u64;
```

Comment:
Do we want to `ceil()` here, and elsewhere we call `round()`?

Reply:
Hrm, actually, reading this and the code below, it weirds me out a little bit that we're calculating this information twice, as far as I can tell? See below, in `proposed_dataset_changes`, seems like we're doing this same calculation again...

maybe this is fine, but it just raised a flag to me -- if we're doing floating point math on accounting, it kinda seems like we should do it "exactly once".

Reply:
95696ef00636f16345ed388457407ffbc76e81bf removes the floating point stuff, and also uses the `size_delta` variable in the spot you identified. nice catch :)